hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -1955,19 +1955,15 @@ public Set<String> getMultipartUploadKeys(
       }
     }
 
+    // prefixed iterator will only iterate until keys match the prefix
     try (TableIterator<String, ? extends KeyValue<String, OmMultipartKeyInfo>>
-        iterator = getMultipartInfoTable().iterator()) {
```

Comment:
i don't see this one became a problem in my stacktrace, but it should be fixed too.

There are 100+ of similar iterator usages. It would be a huge effort to try and update all of them.